mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/DefaultPropertyDefinition.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.internal.mapper.processor.entity;
+
+import com.squareup.javapoet.TypeName;
+
+public class DefaultPropertyDefinition implements PropertyDefinition {
+
+  private final String cqlName;
+  private final String getterName;
+  private final String setterName;
+  private final TypeName type;
+
+  public DefaultPropertyDefinition(
+      String cqlName, String getterName, String setterName, TypeName type) {
+    this.cqlName = cqlName;
+    this.getterName = getterName;
+    this.setterName = setterName;
+    this.type = type;
+  }
+
+  @Override
+  public String getCqlName() {
```

Comment:
Same here, I suppose that we will provide means for CQL name customization in the future?

Reply:
Yes. Quite honestly the whole `PropertyDefinition` API is tentative: it gives us something to build upon for the next tickets, but it will probably evolve over time.